### PR TITLE
Bratseth/bound rendering tasks

### PR DESCRIPTION
**Pull request by bratseth**

### CODE REVIEW

**Review by baldersheim — DISMISSED** [review]
No deadlock potential....

File: container-core/src/main/java/com/yahoo/processing/rendering/AsynchronousSectionedRenderer.java [hunk 1]
```diff
@@ -585,13 +585,13 @@ public void addDataListener(Runnable listener) {
...
         public Data get(int index) {
-            if (index>0) throw new IndexOutOfBoundsException();
+            if (index > 0) throw new IndexOutOfBoundsException();
             return trueTopLevel;
         }
 
         @Override
         public List<Data> asList() {
-            return Collections.<Data>singletonList(trueTopLevel);
+            return Collections.singletonList(trueTopLevel);
```
Comment (baldersheim) [inline container-core/src/main/java/com/yahoo/processing/rendering/AsynchronousSectionedRenderer.java:594]: List.of() ?

**Review by baldersheim — APPROVED** [review]

**Review by bjorncs — COMMENTED** [review]
100 is on the low side

**Comment by bratseth** [comment]
Per core?

**Pull request merged**